src/tests/electron/setup/timeouts.ts

```diff
@@ -2,11 +2,11 @@
 // Licensed under the MIT License.
 
 // How long to allow an entire test to execute, including startup/setup
-export const DEFAULT_ELECTRON_TEST_TIMEOUT_MS = 30000;
+export const DEFAULT_ELECTRON_TEST_TIMEOUT_MS = 60000;
```

Comment:
unrelated to electron update - this loosens some of the values set in https://github.com/microsoft/accessibility-insights-web/pull/2506/files; on my laptop the original values are aggressive enough that the app doesn't usually connect in time.